docs/pages/admin-guides/access-controls/access-request-plugins/datadog-hosted.mdx

```diff
@@ -113,23 +113,23 @@ to the Teleport plugin.
 
 First, in the Datadog web UI navigate to **Organization Settings -> Access -> API Keys**
 to view your organization's API keys. To generate a new API key, click on **New Key**
-and name it "Teleport Access". Copy the API key to paste in a later step.
+and name it "Zero Trust Access". Copy the API key to paste in a later step.
 
 ![Datadog dashboard for API Keys](../../../../img/enterprise/plugins/datadog/dashboard-api-keys.png)
 
 ### Create a Datadog Service Account
 
 Next, navigate to **Organization Settings -> Identity & Accounts -> Service Accounts**
 to view your organization's service accounts. To create a new service account, click
-on **New Service Account** and name it "Teleport Access". You can provide whatever
+on **New Service Account** and name it "Zero Trust Access". You can provide whatever
 email you'd like to associate with the service account, then assign the role
 "Datadog Standard Role". This role provides the required permissions necessary to
-manage the Teleport Access Request incidents.
+manage the Zero Trust Access Request incidents.
 
 ![Datadog dashboard for Service Accounts](../../../../img/enterprise/plugins/datadog/dashboard-service-accounts.png)
 
-Now click on the "Teleport Access" service account that was just created and click
-on the **Application Keys** tab and click on **New Key** and name it "Teleport Access"
+Now click on the "Zero Trust Access" service account that was just created and click
+on the **Application Keys** tab and click on **New Key** and name it "Zero Trust Access"
```

Comment:
Same as above.

This change is a net loss. The original name made it clear that the account you created in Datadog was to be used with Teleport. Now it's just a generic security term that doesn't refer to Teleport in any way.